src/bdscript/onlyBotPerms.md

```diff
@@ -7,5 +7,5 @@ $onlyBotPerms[permissions;errorMessage]
 ```
 
 ### Breakdown
-- `permissions` - The permissions that the bot needs to run the command.
+- `permissions` - The [permissions](/src/resources/permissions.md) that the bot needs to run the command.
```

Comment:
Please correct arguments in the full function
[JSON function data](https://botdesignerdiscord.com/public/api/function/$onlyBotPerms[])